test/runtests.jl

```diff
@@ -15,6 +15,11 @@ const PRETRAINED_MODELS = [
     (ResNet, 152),
 ]
 
+function _gc()
+    GC.safepoint()
+    GC.gc()
```

Comment:
I wonder if we should be more aggressive and do:
```suggestion
    GC.gc(true)
```

Reply:
Ooh, let's try this. The refactor means one line change should work

Reply:
....this killed even the one test that _was_ passing 😭